tests/ossfuzz/Makefile

```diff
@@ -1,6 +1,61 @@
-fuzz: fuzz.c
-	$(CC) $(CFLAGS) -c fuzz.c -I ../../src/liblzma/api/
-	$(CXX) $(CXXFLAGS) $(LIB_FUZZING_ENGINE) fuzz.o -o $(OUT)/fuzz \
+fuzz_decode_xz: fuzz_decode_xz.c
+	$(CC) $(CFLAGS) -c fuzz_decode_xz.c -I ../../src/liblzma/api/
+	$(CXX) $(CXXFLAGS) $(LIB_FUZZING_ENGINE) fuzz_decode_xz.o -o $(OUT)/fuzz_decode_xz \
+	      ../../src/liblzma/.libs/liblzma.a
+
+fuzz_decode_xz_crc: fuzz_decode_xz_crc.c
```

Comment:
This Makefile could be done with wildcards instead of repeating the same commands for each fuzz target since all of the .c/.o files become fuzz targets.